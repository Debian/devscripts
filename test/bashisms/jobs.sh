#!/bin/sh

# POSIX+UP:
jobs # BASHISM
jobs -l # BASHISM
jobs -p # BASHISM

# Non-POSIX at all:

sleep 10 &
j=$(jobs -p) # possible BASHISM (context changes because of subshell)
jobs -r # BASHISM
jobs -s # BASHISM
jobs -n # BASHISM
jobs -x # BASHISM