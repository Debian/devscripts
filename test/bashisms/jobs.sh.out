possible bashism in bashisms/jobs.sh line 4 (jobs):
jobs # BASHISM
possible bashism in bashisms/jobs.sh line 5 (jobs):
jobs -l # BASHISM
possible bashism in bashisms/jobs.sh line 6 (jobs):
jobs -p # BASHISM
possible bashism in bashisms/jobs.sh line 11 (jobs):
j=$(jobs -p) # possible BASHISM (context changes because of subshell)
possible bashism in bashisms/jobs.sh line 12 (jobs):
jobs -r # BASHISM
possible bashism in bashisms/jobs.sh line 13 (jobs):
jobs -s # BASHISM
possible bashism in bashisms/jobs.sh line 14 (jobs):
jobs -n # BASHISM
possible bashism in bashisms/jobs.sh line 15 (jobs):
jobs -x # BASHISM
